spi/common/web-spi/src/main/java/org/eclipse/edc/web/spi/WebServer.java

```diff
@@ -31,4 +34,12 @@ public interface WebServer {
      */
     void addPortMapping(String contextName, int port, String path);
 
+
+    /**
+     * Returns the default context name
+     */
+    default String getDefaultContextName() {
```

Comment:
There was an override from the jetty one but i did inline the default in the interface instead of the impl. I would prefer to leave it it configurable. It may not happen in the near future to have another web server impl but who knows :)